tasks/install.yml

```diff
@@ -91,8 +93,8 @@
   failed_when: False
   when:
     - not gitlab_bootstrapped.stat.exists
-    - gitlab_admin_username|trim != ""
-    - gitlab_admin_password|trim != ""
+    - gitlab_admin_username|trim | length > 0
+    - gitlab_admin_password|trim | length > 0
 
 - name: "Create GitLab SSL configuration folder."
   file:
```

Comment:
Comment for line 111:
why using command openssl call instead of `openssl_certificate` ansible module?